### ISSUE TITLE
Set Powershell ExcutionPolicy for CurrentUser

### ISSUE THREAD

**Issue by subsign** [Chuyu-Team/Dism-Multi-language#481]
For `powershell.exe Set-ExecutionPolicy unrestricted`

HKEY_CURRENT_USER\Software\MicrosoftPowerShell\1\ShellIds\Microsoft.PowerShell

**Comment by verdy-p** [comment]
I would not recommend this unrestricted setting, which is in fact not needed for most users and will not improve at al lthe system performance; even if this is not defined, users will get an authorization request from the system, or the access will be silently denied and the application needing it will display an error. Users should be aware about which app needs that. But most apps now do not need that, if they are properly signed. Only a few application developers may need that as a convenience for the software they are currently devlopping or prototyping, but in fine, an app ready to deploy should no longer need that.
You also don't need that setting for running local administration: if your maintenance script fails, just run it from a temporary elevated administrator "cmd". For installing services that would need such elevation it should be done in a specific administrator account for that service; otherwise use the powershell script which will first use explicitly the "Set-ExecutionPolicy" for itself and it will not survive your current Powershell session
For now I've never seen really any app or service or regular maintenance script that requires this, not even for using Dism++ itself.
However this setting may be checked by Dism, so that it will detect if it is used and will suggest removing it. Some existing antimalware tools already do that and will help diagnose and secure your system: don't use that "unrestricted" policy in a permanent global setting in the registry.

**Comment by 12donkeys** [comment]
@verdy-p I agree 100%. I was actually just contributing to the pool of options that are available to incorporate. You're most certainly correct though!
